assets/base.css

```diff
@@ -307,6 +307,12 @@ body,
   margin: 0 auto 2rem;
 }
 
+@media screen and (min-width: 990px) {
+  .grid-full-width .rte-width {
```

Comment:
Don't want to add too much noise, but as I mentioned above, for readability purposes I think a better approach to this may be to remove the padding on the rte-width instead of removing the restricted width. This of course may become slightly more complicated given the text-align left/right options on card.

Reply:
I'll confirm with @wiktoriaswiecicka before if we want to do that or center it instead 🤔 
Context: https://github.com/Shopify/dawn/pull/989#discussion_r800882914